components/common/SideBar.tsx

```
@@ -1,40 +1,302 @@
+						{/* Sidebar component, swap this element with another sidebar if you like */}
+						<div className="flex grow flex-col gap-y-5 overflow-y-auto bg-gradient-to-b from-amber-100 to-orange-200 px-6 pb-2">
+							<div className="flex h-16 shrink-0 items-center justify-center">
+								<Link
+									href="/rooms"
+									className="text-4xl font-bold text-orange-800 hover:text-orange-600 font-cherry"
+								>
+									ALBUM
+								</Link>
+							</div>
+							<nav className="flex flex-1 flex-col">
+								<ul role="list" className="flex flex-1 flex-col gap-y-7">
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<ol>
<ul>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)